### ISSUE TITLE
Installer issue 

### ISSUE THREAD

**Issue by gajananalienhu** [GanuM/profitGuruPOS#724]
while installing application its showing error automatically its not deleting Profit Guru A folder 
error unable to delete Profit Guru A folder

**Comment by saichaitanyaeruvaram** [comment]
@ajaykumarpatil  
Test on different PCs and update me.
windows -7,8,8.1,10 - uninstall/install.. install without uninstalling

**Comment by ajaykumarpatil** [comment]
Yeah Anna i go through Windows 7 and 10 its working fine in both version , once i have to check with windows 8.
